common/scripts/x86_64/Ubuntu_14.04/_helpers.sh

```diff
@@ -77,17 +82,18 @@ __check_dependencies() {
   ################## Install Docker  #####################################
   if type docker &> /dev/null && true; then
     __process_msg "'docker' already installed, checking version"
-    local docker_version=$(docker --version)
-    if [[ "$docker_version" == *"$DOCKER_VERSION"* ]]; then
-      __process_msg "'docker' $docker_version installed"
+    local min_docker_version=1.13
+    local docker_version=$(docker version --format '{{.Server.Version}}')
+    if __version_gte "$docker_version" "$min_docker_version"; then
+      __process_msg "'docker' $docker_version version is greater than equal to $min_docker_version"
     else
-      __process_error "Docker version $docker_version installed, required $DOCKER_VERSION"
+      __process_error "Docker version $docker_version installed, min version required = $min_docker_version"
       __process_error "Install Docker Version $DOCKER_VERSION from \"
-      https://docs.docker.com/cs-engine/1.13/#install-on-ubuntu-1404-lts-or-1604-lts"
+      https://docs.docker.com/v17.06/engine/installation/linux/ "
       exit 1
```

Comment:
- will this not exit?
- also, according to the design, we need to set the docker version in the ENV and then use that env for any comparisions. this implementation is incorrect
- secondly, why are we complicating this? if the installed version is not equal to desired version, just print a message and proceed. not sure why we need less than/greater than check here